src/rev_ai/apiclient.py

```diff
@@ -209,6 +219,11 @@ def get_transcript_json(self, id_):
             url, headers={'Accept': self.rev_json_content_type})
         response.raise_for_status()
 
+        if filename:
+            path = os.path.join(filepath, filename+'.json')
+            with open(path,'w+') as f:
+                json.dump(response.json(), f)
```

Comment:
The file that comes from this way of writing isn't nicely formatted json (ie. no spacing or indents). We can fix this by calling `json.dump(response.json(), f, indent=4)`